openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/MetadataToESConfigForm/MetadataToESConfigFieldLabel.component.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { Space, Typography } from 'antd';
+import React from 'react';
+
+const { Text } = Typography;
+
+export default function GetFieldLabels({
```

Comment:
```suggestion
export default function EsConfigFieldLabels({
```

Reply:
Done

Reply:
@Sachin-chaurasiya I've made the suggested changes. Can you please review